src/routes/blogs/accelerating-llama-2/+page.svelte

```diff
@@ -18,11 +18,39 @@
 		name="description"
 		content="Explore how ONNX Runtime can propel your Llama2 variants for faster inference."
 	/>
+	<meta
+		name="keywords"
+		content="Accelerating LLaMA-2, ONNX Runtime, Inference, AI, Microsoft, Meta, Llama2, Performance Optimization, Multi-GPU Inference"
+	/>
+	<meta name="author" content="Kunal Vaishnavi, Parinita Rahi" />
+	<meta name="date" content="2023-11-14" />
+	<meta property="og:title" content="Accelerating LLaMA-2 Inference with ONNX Runtime" />
+	<meta
+		property="og:description"
+		content="Explore how ONNX Runtime accelerates LLaMA-2 inference, achieving up to 3.8X faster performance for models ranging from 7B to 70B parameters. Learn about graph fusions, kernel optimizations, multi-GPU inference support, and more."
+	/>
+	<meta property="og:type" content="article" />
+	<meta property="og:url" content="https://onnxruntime.ai/blogs/accelerating-llama-2" />
+	<meta property="og:image" content={figure5} />
+	<meta property="og:site_name" content="ONNX Runtime" />
+	<meta name="twitter:card" content={figure5} />
+	<meta name="twitter:title" content="Accelerating LLaMA-2 Inference with ONNX Runtime" />
+	<meta
+		name="twitter:description"
+		content="Explore how ONNX Runtime can propel your Llama2 variants for faster inference."
+	/>
+	<meta name="twitter:image" content={figure5} />
 </svelte:head>
 <Header pathvar="" />
 <div class="container mx-auto px-4 md:px-8 lg:px-48 pt-8">
 	<h1 class="text-5xl pb-2">Accelerating LLaMA-2 Inference with ONNX Runtime</h1>
-	<p class="text-neutral">By: <a href="https://www.linkedin.com/in/kunal-v-16315b94" class="text-blue-500">Kunal Vaishnavi</a> and <a href="https://www.linkedin.com/in/parinitaparinita/" class="text-blue-500">Parinita Rahi</a> </p>
+	<p class="text-neutral">
+		By: <a href="https://www.linkedin.com/in/kunal-v-16315b94" class="text-blue-500"
+			>Kunal Vaishnavi</a
+		>
+		and
+		<a href="https://www.linkedin.com/in/parinitaparinita/" class="text-blue-500">Parinita Rahi</a>
+	</p>
 	<p class="text-neutral">14TH NOVEMBER, 2023</p>
```

Comment:
seems like we should add a note that the blog was updated on 11/22 with revised numbers

Reply:
Updated.